docs/manuals/source/Appendix/Troubleshooting.rst

```diff
@@ -127,19 +132,19 @@ Below is a super stripped down :file:`bareos-dir.conf` file showing you the four
    # Bareos Director Configuration file -- bareos-dir.conf
```

Comment:
```suggestion
   # Bareos Director Configuration /etc/bareos/bareos-dir.d/*
```

Reply:
I will also change the preceding paragraph.